region/maridia/inner-green.json

```diff
@@ -1269,6 +1327,43 @@
                     "HiJump",
                     "canPlayInSand"
                   ]
+                },
+                {
+                  "name": "Cross Room Jump",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    "canPlayInSand",
+                    {"adjacentRunway": {
+                      "fromNode": 2,
+                      "usedTiles": 1,
+                      "physics": ["air"]
+                    }}
+                  ],
+                  "note": [
+                    "Requires a runway of only one tile in the adjacent room.",
```

Comment:
I get sniped trying this.
You need another item to advance from the tall pillar.
Maybe this could jump far enough with my runway / speed.